modules/services/mapilio.js

```diff
@@ -348,25 +362,39 @@ export default {
         if (!d) return this;
 
         let wrap = context.container().select('.photoviewer .mapilio-wrapper');
-        let attribution = wrap.selectAll('.photo-attribution').text('');
-
-        if (d.capture_time) {
+        let attribution = wrap.selectAll('.photo-attribution').text('\u00A0');
+      
+        getUserData(d.created_by_id).then((username) => {
+          if (username) {
             attribution
-                .append('span')
-                .attr('class', 'captured_at')
-                .text(localeDateString(d.capture_time));
-
+              .append('a')
+              .attr('class', 'captured_by')
+              .attr('target', '_blank')
+              .text('@' + username);
```

Comment:
this is an `<a>` element, but it has no `href`. Is there a way to link to a mapillio user profile? If not, we should remove this link

Reply:
thanks!